src/features/houseHold/components/HouseHoldForm.tsx

```diff
@@ -0,0 +1,115 @@
+"use client";
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { z } from "zod";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+import { houseHoldSchema } from "../schema/houseHolds";
+import { insertHouseHold } from "../actions/houseHolds";
+import { useTranslations } from "next-intl";
+
+export function HouseHoldForm({
+  currencies,
+  locale,
+}: {
+  currencies: { code: string }[];
```

Comment:
wiem że to ograniczenie typowo nexta ale kiedyś w przyszłości będzie można się pokusić na przekazywanie currencies w jakiś inny sposób

może kontekstem

może przekazywać cały currencypicker jako server component